bridges/BinanceBridge.php

```diff
@@ -0,0 +1,105 @@
+<?php
+class BinanceBridge extends BridgeAbstract {
+	const NAME = 'Binance';
+	const URI = 'https://www.binance.com';
+	const DESCRIPTION = 'Subscribe to the Binance blog or the Binance Zendesk announcements.';
+	const MAINTAINER = 'thefranke';
+	const CACHE_TIMEOUT = 0; // 1h
```

Comment:
Please provide a value for the cache.
BTW, while coding the bridge you can enable [debug mode](https://github.com/RSS-Bridge/rss-bridge/wiki/Debug-mode) that ignores totally the cache regardless of the `CACHE_TIMEOUT` value.

Reply:
Damnit! Thanks for the hint, I'll use this from now on, resetting the value is too easy to forget.